packages/builder/src/components/design/settings/controls/ButtonConfiguration/ButtonConfiguration.svelte

```diff
@@ -0,0 +1,134 @@
+<script>
+  import DraggableList from "../DraggableList/DraggableList.svelte"
+  import ButtonSetting from "./ButtonSetting.svelte"
+  import { createEventDispatcher } from "svelte"
+  import { store } from "builderStore"
+  import { Helpers } from "@budibase/bbui"
+
+  export let componentBindings
+  export let bindings
```

Comment:
We can consume bindings and componentBindings as a prop rather than having to generate them ourselves, so I removed them below and just use them here 👌